diff --git a/node_modules/framer-motion/dist/index.d.ts b/node_modules/framer-motion/dist/index.d.ts
index d5aa9c1..ccf421c 100644
--- a/node_modules/framer-motion/dist/index.d.ts
+++ b/node_modules/framer-motion/dist/index.d.ts
@@ -1900,6 +1900,7 @@ interface AnimationPlaybackControls {
     attachTimeline?: (timeline: ProgressTimeline) => VoidFunction;
 }
 declare type DynamicOption<T> = (i: number, total: number) => T;
+// @ts-expect-error
 interface CSSStyleDeclarationWithTransform extends Omit<CSSStyleDeclaration, "direction" | "transition"> {
     x: number | string;
     y: number | string;
