diff --git a/node_modules/import-jsx/index.js b/node_modules/import-jsx/index.js
index 30dea40..c5ef993 100644
--- a/node_modules/import-jsx/index.js
+++ b/node_modules/import-jsx/index.js
@@ -3,7 +3,7 @@ import cachedTransform, {cacheKeyFromSource} from './cache.js';
 import transform from './transform.js';
 
 export const load = async (url, _context, nextLoad) => {
-	if (!url.endsWith('.js') || url.includes('node_modules')) {
+	if (!(url.endsWith('.js') || url.endsWith('.mjs')) || url.includes('node_modules')) {
 		return nextLoad(url);
 	}
 
