diff --git a/node_modules/just-order-by/index.cjs b/node_modules/just-order-by/index.cjs
index 2bebe1b..685f888 100644
--- a/node_modules/just-order-by/index.cjs
+++ b/node_modules/just-order-by/index.cjs
@@ -5,11 +5,8 @@ function getValueFromParam(param, item) {
     return param.property(item);
   }
 
-  if (typeof param.property === 'string') {
-    return item[param.property];
-  }
-
-  throw new TypeError('Type of `property` property should be a string or a function');
+  // Allow param.property to be a number, such as if item is an array rather than a normal object
+  return item[param.property];
 }
 
 function createSortFn(params) {
