diff --git a/types/index.d.ts b/types/index.d.ts
index 770a77179b88bacf5ad9283ae92975d24581d108..5934346cab2850a003869615ab0cf4a4be767131 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -105,7 +105,6 @@ export type WrappedComponentFactory<P> = (props: P) => JSX.Element;
 
 export type WrappedComponent<P> =
   | React.ComponentClass<P>
-  | React.SFC<P>
   | WrappedComponentFactory<P>;
 
 export function SortableContainer<P>(
diff --git a/yarn.lock b/yarn.lock
deleted file mode 100644
index c6f4409d49066905c78cf9baaa361d851bc98512..0000000000000000000000000000000000000000
