diff --git a/node_modules/react-sortable-hoc/types/index.d.ts b/node_modules/react-sortable-hoc/types/index.d.ts
index 770a771..5934346 100644
--- a/node_modules/react-sortable-hoc/types/index.d.ts
+++ b/node_modules/react-sortable-hoc/types/index.d.ts
@@ -105,7 +105,6 @@ export type WrappedComponentFactory<P> = (props: P) => JSX.Element;
 
 export type WrappedComponent<P> =
   | React.ComponentClass<P>
-  | React.SFC<P>
   | WrappedComponentFactory<P>;
 
 export function SortableContainer<P>(
