diff --git a/node_modules/resize-observer-polyfill/src/index.d.ts b/node_modules/resize-observer-polyfill/src/index.d.ts
index 74aacc0..ddfee00 100644
--- a/node_modules/resize-observer-polyfill/src/index.d.ts
+++ b/node_modules/resize-observer-polyfill/src/index.d.ts
@@ -16,7 +16,7 @@ declare global {
 
     interface ResizeObserverEntry {
         readonly target: Element;
-        readonly contentRect: DOMRectReadOnly;
+        // readonly contentRect: DOMRectReadOnly;
     }
 
     interface ResizeObserver {
